tfac/dataImport.py

```diff
@@ -130,6 +131,9 @@ def add_missing_columns(data, patients):
         data (pandas.DataFrame): cytokine/RNA data with missing columns
             added; sorted by patient numbers
     """
+    shared = set(data.columns) & set(patients)
```

Comment:
Is this needed?

Reply:
Yup--in removing the RNA-only patients, we'll have samples that appear in the data but not in the patient labels. Unsure if this is the best place to do this, though.

Reply:
Oh! Ok, just make the comment clearer that this removes patient labels that are missing corresponding data.